### PR TITLE
Feat/reorganization

### PR DESCRIPTION
**Pull request by Eccleria**
Major modifications to the global files organization, rename files, add some commentaries, readme draft

### CODE REVIEW

File: src/commands/personality.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import personalities from "../jsons/personalities.json";
+
+export var PERSONALITY = personalities.normal;
```
Comment (Eccleria) [inline src/commands/personality.js:3]: Change var into something else

Comment (Eccleria) [inline src/commands/personality.js:3]: Change var into something else

File: src/commands/spotify.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+  const foundLink = await parseLink(
+    message.content,
+    client,
+    PERSONALITY.spotify,
+    currentServer
+  );
+
+  await spotifyReply(foundLink, message, client, currentServer);
+};
+
```
Comment (Eccleria) [inline src/commands/spotify.js:15]: Decide if the spotifyHelper.js file is really usefull or should be merged with spotify.js file.

Comment (Eccleria) [inline src/commands/spotify.js:15]: Decide if the spotifyHelper.js file is really usefull or should be merged with spotify.js file.

**Review by Titch88 — APPROVED** [review]
LETS GO

**Pull request merged**